docker-compose.yaml

```
@@ -0,0 +1,37 @@
+version: '3'
+services:
+  kubearmor-init:
+    image: kubearmor/kubearmor-init:stable
+    volumes:
+      - /opt/kubearmor/BPF:/opt/kubearmor/BPF
+   
+
+  kubearmor:
+    image: kubearmor/kubearmor:latest
+    volumes:
+      - /opt/kubearmor/BPF:/opt/kubearmor/BPF
+      - /sys/fs/bpf:/sys/fs/bpf
+      - /sys/kernel/security:/sys/kernel/security
+      - /sys/kernel/debug:/sys/kernel/debug
+      - /var/run/containerd/containerd.sock:/var/run/containerd/containerd.sock
+      - /run/containerd:/run/containerd
+      - /var/lib/docker:/var/lib/docker
+    privileged: true
+    cap_add:
+      - SETUID
+      - SETGID
+      - SETPCAP
+      - SYS_ADMIN
+      - SYS_PTRACE
+      - MAC_ADMIN
+      - SYS_RESOURCE
+      - IPC_LOCK
+      - CAP_DAC_OVERRIDE
+      - CAP_DAC_READ_SEARCH
+    pid: host
+    ipc: host
+    network_mode: host
+    depends_on:
+      kubearmor-init:
+        condition: service_healthy # container will wait for the kubearmor-init container 
```

```suggestion
        condition: service_completed_successfully # container will wait for the kubearmor-init container 
```
> service_completed_successfully. This specifies that a dependency is expected to run to successful completion before starting a dependent service.

We should use this condition instead
https://docs.docker.com/compose/startup-order/#control-startup

---

hey @daemon1024 sorry for the delay , i was too busy ( due to exams ) , here is the update  - 

- i don't know what is the problem with `- /sys/kernel/security:/sys/kernel/security` volume it give me 
` Error response from daemon: mkdir /sys/kernel/security: operation not permitted` while  running the `kubearmor` container 
- else when excluding the above volume its running fine 
- and i have verified policy enforcement and alerts after passing `-k8s=false` and your above suggested change as well .

please guide me in this and there wont be any delay from now